libraries/botbuilder-ai/src/languageTranslator.ts

```diff
@@ -14,7 +14,8 @@ export interface TranslatorSettings {
     nativeLanguages: string[],
     noTranslatePatterns: Set<string>,
     getUserLanguage?: ((c: TurnContext) => string) | undefined,
```

Comment:
>c: TurnContext) => string [](start = 24, length = 25)

These signatures should be more consistant.  use `context` instead of `c` for instance.